backend/src/email/routes/email-campaign.routes.ts

```diff
@@ -27,6 +27,12 @@ const storeTemplateValidator = {
       .lowercase()
       .allow(null)
       .required(),
+    from: Joi.string()
```

Comment:
Remember I wanted to make this a default / optional because we need to maintain backwards compatibility?

Reply:
Oh ya ok now I remember will make the changes